src/tools/quemap/map.c

```diff
@@ -769,6 +808,8 @@ static entity_t *ParseEntity(parser_t *parser) {
 			Com_Error(ERROR_FATAL, "MAX_BSP_ENTITIES\n");
 		}
 
+		memset(plane_hash, 0, sizeof(plane_hash));
+
```

Comment:
Were it not for the bevel nonsense, this line alone would have been enough to guarantee unique planes for each inline model, heh.